lxc/copy.go

```diff
@@ -236,6 +248,11 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 			return err
 		}
 
+		// Source container needs to be stopped when using the refresh option
+		if c.flagRefresh && entry.Status == "Running" {
```

Comment:
Hmm, actually, this feature becomes a lot less useful if the source container has to be stopped, what's the reason for this?

That the destination container must be stopped makes a lot of sense though. This kind of check should also typically be done server side rather than in the client.